test-data/unit/lib-stub/attr.pyi

```diff
@@ -87,8 +95,12 @@ def attrs(maybe_cls: None = ...,
           init: bool = ...,
           slots: bool = ...,
           frozen: bool = ...,
+          weakref_slot: bool = ...,
           str: bool = ...,
-          auto_attribs: bool = ...) -> Callable[[_C], _C]: ...
+          auto_attribs: bool = ...,
+          kw_only: bool = ...,
+          cache_hash: bool = ...,
```

Comment:
I noticed that some of these are missing in typeshed. Could you please make a typeshed update PR?